roles/tpa_single_node/templates/manifests/importer/DeploymentA.yaml.j2

```diff
@@ -1,135 +0,0 @@
-apiVersion: apps/v1
```

Comment:
Please restore and optimize the configuration in a separate PR, there are missing placeholder in the Deployment.yaml.j2

Reply:
see above